Makefile

```diff
@@ -88,7 +88,7 @@ endif
 ## Ensures NPM dependencies are installed without having to run this all the time.
 webapp/node_modules: $(wildcard webapp/package.json)
 ifneq ($(HAS_WEBAPP),)
-	cd webapp && $(NPM) install
+	cd webapp && $(NPM) install --verbose
```

Comment:
That seems like an unrelated change

Reply:
hmm, that was committed by @mickmister in  [##30c35d2](https://github.com/mattermost/mattermost-plugin-gitlab/pull/307/commits/30c35d2c93b0ad2849ea40c754bcf20789141797) I don't remember anything about this PR that would be related to that.

Reply:
This was put here due to something related to CI, but that has since been fixed. We should make sure we're not re-introducing this change here. To be clear `--verbose` should not be used

Reply:
OK, I reverted it